games-util/pyfa/pyfa-1.35.2.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="sqlite,threads"
+
+inherit eutils desktop python-r1 python-utils-r1
+
+DESCRIPTION="Python Fitting Assistant - a ship fitting application for EVE Online"
+HOMEPAGE="https://github.com/pyfa-org/Pyfa"
+
+LICENSE="GPL-3+ LGPL-2.1+ CC-BY-2.5 free-noncomm"
```

Comment:
I'm sorry if you've been asked about this already but do you actually need all those licenses simultaneously?

Reply:
You are absolutely right, I don't think they are correct. The software itself is GPL-3, but there are game assets bundled with the code which have another licenses and it is not clear to me what they are. I opened a ticked about it upstream and try to investigate this. May take some time.

Reply:
If those assets are installed and are indeed licensed with all those licenses, then `LICENSE` is correct.

Reply:
Ok, sorry for the long delay. The developer just got an answer from the asset owners and it's more or less "we don't know". https://github.com/pyfa-org/Pyfa/issues/1505

What he proposes is that we include the text snipped he has on the bottom of the README here: https://github.com/pyfa-org/Pyfa/blob/master/README.md

So should I just add that text as a licence file and add that that as license together with GPL3 for the pyfa code? Is there a naming convention for this (I looked around but found none)?

CCP-assets-pfya (because the text is granting the rights to pyfa specifically) ?

Reply:
Let's wait for @gentoo/licenses to advise. In the meantime, please implement all the remaining remarks.

Reply:
I have already commented at https://github.com/pyfa-org/Pyfa/issues/1505 but I must admit that I am somewhat at a loss concerning the license situation.

Especially, is the text below "License" at https://community.eveonline.com/community/fansites/toolkit/ a statement from the assets' copyright holder? To whom is it addressed (i.e., who is "you" in that text)? Note that the only clause that grants any rights is "we will give you a license to use them". All the rest of that text are conditions imposed on "you" restricting that usage right. There is no clause that would grant the right to redistribute.

My current conclusion would be that LICENSE needs to include "all-rights-reserved" (because it certainly isn't "free-noncomm") along with RESTRICT="mirror bindist".

I would also suggest that you file a Gentoo bug (with licenses team in CC or as assignee) if you want to further discuss the licensing situation, so that we have a traceable record of any decisions taken.